includes/class-wcpdf-install.php

```diff
@@ -347,6 +347,17 @@ protected function upgrade( $installed_version ) {
 			update_option( 'wpo_wcpdf_settings_debug', $debug_settings );
 		}
 
+		// 2.8.2: rename invoice number store table
+		if ( $installed_version == 'versionless' || version_compare( $installed_version, '2.8.2', '<=' ) ) {
+			global $wpdb;
+			$legacy_table_name = "{$wpdb->prefix}wcpdf_invoice_number";
+			$current_year      = date('Y');
+			$new_table_name    = "{$wpdb->prefix}wcpdf_invoice_number_{$current_year}";
```

Comment:
table name could have been filtered:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/05efc0fa692a0a75a17645744d6c1e671239f3aa/includes/documents/class-wcpdf-sequential-number-store.php#L42
We should include this scenario in tests, looking up code snippets that we have provided to users in the past.